lib/message.js

```diff
@@ -0,0 +1,71 @@
+import { log } from '../log';
+import { chooseRandom } from '../lib/rng';
+import * as dearViktor from '../data/dearviktor.json';
+import * as commands from '../data/commands.json';
+
+const isChannelDM = msg => msg.author.id === msg.channel.id;
+
+const isUserAdmin = msg => msg.member.hasPermission(checkAdmin);
+const isUserBot = msg => msg.author.bot;
+const isUserArcy = msg => msg.author.id === '165962236009906176';
+
+const messageStartsWithCommandSymbol = msg => msg.content.startsWith(commands.options.commandSymbol);
+
+const isMessageRant = msg => msg.content === msg.content.toUpperCase() && msg.content.length > 20;
+const isMessageCommand = msg => commands.list.find(cmd => cmd.keyword === message.getKeyword(msg));
+const isMessageDearViktor = msg => msg.content.toLowerCase().startsWith('dear viktor');
+const isMessageDearVictor = msg => msg.content.toLowerCase().startsWith('dear victor');
+
+export const message = {
+    classify: msg => {
+        if (isMessageRant(msg))
+            msg.react('🍿');
+
+        if (isChannelDM(msg) && !isUserArcy(msg))
+            return message.answer(msg, 'Only my glorious creator can talk to me in private.');
+        if (isMessageDearViktor(msg))
+            return message.answerDearViktor(msg);
+        if (isMessageDearVictor(msg))
+            return message.answerDearVictor(msg);
+
+        if (messageStartsWithCommandSymbol(msg)) 
```

Comment:
This should actually be checked first. At the moment you can't get the help via DM as it denies communication

Reply:
It's intended. Original Vikbot version blocks any commands from being executed in DMs. I don't remember why it was disabled but it was for a reason. I think it was linked to him being customized by guild and DMs causing some errors regarding that. For now I want it to be disabled till I get to the point of customizing him by guilds.

Reply:
Then at least it should move above the fluff checks.